ui/app/js/controllers/rules_controller.js

```diff
@@ -10,7 +10,8 @@ function($scope, $routeParams, $location, $timeout, Rules, Search, $modal, $rout
   $scope.pr_ch_options = [];
 
   $scope.currentPage = 1;
-  $scope.pageSize = 20;
+  $scope.pagesize_options = [10, 25, 50, 100]
```

Comment:
I'm surprised it doesn't cause a test failure (or a failure in actual usage), but this is throwing an error:
```
balrogui_1     |    app/js/controllers/rules_controller.js
balrogui_1     |      13 |  $scope.pagesize_options = [10, 25, 50, 100]
balrogui_1     |                                                       ^ Missing semicolon.
balrogui_1     | 

```